packages/compass-e2e-tests/helpers/selectors.ts

```diff
@@ -550,16 +550,22 @@ export const incompleteSchemaForClonedDocMsg =
 export const ImportModal = '[data-testid="import-modal"]';
 export const ImportDelimiterSelect = '[data-testid="import-delimiter-select"]';
 export const ImportDelimiterMenu = '[id="import-delimiter-select-menu"]';
+export const ImportStopOnErrorsCheckbox =
+  '[data-testid="import-stop-on-errors"]';
 export const ImportFileInput = '#import-file_file_input';
 export const FileTypeJSON = '[data-testid="select-file-type-json"]';
 export const FileTypeCSV = '[data-testid="select-file-type-csv"]';
 export const ImportConfirm =
   '[data-testid="import-modal"] [data-testid="import-button"]';
-export const ImportCancel =
-  '[data-testid="import-modal"] [data-testid="cancel-button"]';
-export const ImportDone =
-  '[data-testid="import-modal"] [data-testid="done-button"]';
-export const ImportErrorBox = '[data-testid="import-error-box"]';
+export const ImportInProgressToast = '[data-testid="import-toast-in-progress"]';
+export const ImportSucceededToast =
+  '[data-testid="import-toast-result-completed"]';
+export const ImportCompletedWithErrorsToast =
+  '[data-testid="import-toast-result-errors-completed"]';
+export const ImportAbortedToast = '[data-testid="import-toast-result-aborted"]';
+export const ImportFailedToast = '[data-testid="import-toast-result-failed"]';
+export const ImportToastAbort =
```

Comment:
Do you need these specific testids if you are correctly checking for text inside the toast in tests already?

Reply:
Removed the extra selectors for variants 👍 definitely cleaner now, nice.